doxygen/doxygen.bzl

```diff
@@ -18,37 +18,36 @@ def _swift_doxygen_impl(ctx):
     doxygen_out = ctx.actions.declare_directory(ctx.attr.name + "_doxygen")
     vars["DOXYGEN_OUTPUT_DIRECTORY"] = doxygen_out.path
 
+    # this performs a CMake-like replacement of @VAR@ based on the vars dict
     config = configure_file_impl(ctx, vars, ctx.attr.name + "_Doxyfile")[0].files.to_list()[0]
 
-    input_dir = ""
-    if "PROJECT_SOURCE_DIR" in vars:
-        input_dir = vars["PROJECT_SOURCE_DIR"]
-
-    project_name = ""
-    if "PROJECT_NAME" in vars:
-        project_name = vars["PROJECT_NAME"]
-
```

Comment:
Note: these are now directly exposed as env variable in bash by setting `env = vars` below.